test/interface/test_filters.py

```diff
@@ -1151,3 +1165,36 @@ def test_complex_using_tuples(self, session):
 
         assert len(result) == 1
         assert result[0].id == 3
+
+
+class TestApplyArrayFilters:
+
+    @pytest.mark.usefixtures('multiple_corges_inserted')
+    def test_value_in_array(self, session, is_postgresql):
```

Comment:
We could rename this test to something like `test_any_value_in_array` to reflect the operator's action.

Reply:
fixed with b0e1b4f